specs/Extensions.md

```diff
@@ -0,0 +1,579 @@
+# Background
+Developers would like to ensure a browser extension is installed so that they may take advantage of the functionality the extension is providing.
+
+# Description
+Enable extension services in WebView2. Then end developer can `AddBrowserExtension` from folder path, `GetBrowserExtension` to get list of extensions installed. Once an extension is installed,
+developers can use `ICoreWebView2BrowserExtension` to get id, name of this extension. And also remove or enable/disable this extension.
+
+# Examples
+
+The following code snippet demonstrates how the Extensions related API can be used:
+
+## Install extension on first run
+
+You may have a browser extension that is specific to your application's web content. This snippet shows how you can install your browser extension on your first run.
+
+### Win32 C++
+```cpp
+static constexpr WCHAR m_defaultExtensionFolderPath = L"//Mydrive";
```

Comment:
What does '//Mydrive' mean? Is this a placeholder for the folder path? If so it would be good to have a normal looking path here and have a comment after to say this needs to be changed for the correct folder. Although if this is supposed to work in our sample app it probably cannot be a hardcoded string and has to be built up to find the assets or whatever folder of the app.